packages/lodestar/src/network/peers/utils/prioritizePeers.ts

```diff
@@ -0,0 +1,84 @@
+import PeerId from "peer-id";
+import {phase0} from "@chainsafe/lodestar-types";
+import {shuffle} from "../../../util/shuffle";
+import {sortBy} from "../../../util/sortBy";
+import {Discv5Query} from "../discover";
+
+/** Target number of peers we'd like to have connected to a given long-lived subnet */
+const MAX_TARGET_SUBNET_PEERS = 6;
+
+/**
+ * Prioritize which peers to disconect and which to connect. Conditions:
+ * - Reach `targetPeers`
+ * - Don't exceed `maxPeers`
+ * - Ensure there are enough peers per active subnet
+ * - Prioritize peers with good score
+ */
+export function prioritizePeers(
+  connectedPeers: {id: PeerId; attnets: phase0.AttestationSubnets; score: number}[],
+  activeSubnetIds: number[],
+  {targetPeers, maxPeers}: {targetPeers: number; maxPeers: number}
+): {peersToDisconnect: PeerId[]; peersToConnect: number; discv5Queries: Discv5Query[]} {
+  const peersToDisconnect: PeerId[] = [];
+  let peersToConnect = 0;
+  const discv5Queries: Discv5Query[] = [];
+
+  // Dynamically compute 1 >= TARGET_PEERS_PER_SUBNET <= MAX_TARGET_SUBNET_PEERS
```

Comment:
assuming this should be 
```suggestion
  // Dynamically compute 1 <= TARGET_PEERS_PER_SUBNET <= MAX_TARGET_SUBNET_PEERS
```